### PR TITLE
Block: Rewards footer #71

### LINKED ISSUE #71: Block: Rewards footer
"Learn more" at the bottom of most pages:
Screenshot 2023-03-15 at 1 52 56 PM

### PR DESCRIPTION
**Pull request by arumsey**
Fix #71

## 🔗 Test URLs:
  
- Before: https://main--caesars--hlxsites.hlx.page/caesars-palace/drafts/arumsey/footer-test
- After: https://issue-71-rewardsFooter--caesars--hlxsites.hlx.page/caesars-palace/drafts/arumsey/footer-test

## 📝 Description:
  
- add ability for footer block to optionally load rewards block as a fragment

### CODE REVIEW

**Comment by tmathern** [comment]
@arumsey I moved the PR out of `draft` into `ready for review, since you asked for reviews.

**Comment by tmathern** [comment]
I thought the rewards cards were meant to be part of that rewards footer? (Even in the DOM there is no rewards cat content yet)

**Comment by arumsey** [comment]
> I thought the rewards cards were meant to be part of that rewards footer? (Even in the DOM there is no rewards cat content yet)

This is part 1 of the issue which builds the ability to conditionally embed a `rewards` fragment into the footer. Once this has been compelted the `rewards.docx` page can be authored to include the actual rewards content. I wanted this PR to only focus on the fragment loading capabilities.

File: caesars-palace/blocks/footer/footer.js [hunk 1]
```diff
@@ -9,72 +15,102 @@ export default async function decorate(block) {
   const cfg = readBlockConfig(block);
   block.textContent = '';
 
+  let resp;
+
+  // add rewards
```
Comment (arumsey) [inline caesars-palace/blocks/footer/footer.js:20]: conditionally embed content from rewards doc unless the `rewards` metadata has been set to `false`.

**Comment by arumsey** [comment]
@tmathern This PR can be merged ahead of rewards content being created. I am hoping the rewards content will be able to use the slider block being worked on my @swetabar.

**Review by tmathern — APPROVED** [review]

**Pull request merged**